app/subscriber/src/components/content-list/utils/groupContent.ts

```diff
@@ -49,7 +49,7 @@ export const groupContent = (groupBy: IGroupByState, content: IContentSearchResu
           const date = new Date(item.publishedOn);
           const hours = date.getHours().toString().padStart(2, '0');
           const minutes = date.getMinutes().toString().padStart(2, '0');
-          key = `${moment(date).format('DD/MM/YY')} (${hours}:${minutes})`;
+          key = `${moment(date).format('DD-MMM-YYYY')} ${`(${hours}:${minutes})`}`;
```

Comment:
Why not `format('DD-MMM-YYYY hh:mm')`?

Reply:
Just in case we eventually add an option to remove time, or if certain content types don't need time when grouped by content. I know it was that way at one point so I just left it easy to change!